demos/demo_finger_forward_kinematics.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""Forward kinematic example using pinocchio
+
+Sends zero-torque commands to the robot and prints finger tip position.
+"""
+import os
+import numpy as np
+import rospkg
+import pinocchio
+import robot_interfaces
+import robot_fingers
+
+if __name__ == "__main__":
+    urdf_pkg_path = rospkg.RosPack().get_path("robot_properties_manipulator")
+    urdf_path = os.path.join(urdf_pkg_path, "urdf", "finger.urdf")
+
+    model = pinocchio.buildModelFromUrdf(urdf_path)
+    data = model.createData()
+    tip_link_id = model.getFrameId("finger_tip_link")
+
+    robot = robot_fingers.Robot(robot_interfaces.finger,
+                                robot_fingers.create_real_finger_backend,
+                                "finger.yml")
+    robot.initialize()
+
+    action = robot.Action()
+    while True:
+        t = robot.frontend.append_desired_action(action)
+        joint_positions = robot.frontend.get_observation(t).position
+
+        # compute the forward kinematics
+        pinocchio.framesForwardKinematics(model, data, joint_positions)
+
+        # get the position of the tip link
+        pos = data.oMf[tip_link_id].translation
```

Comment:
are you sure this is the right frame transformation? i am not an expert with pinocchio, but i remember that @hammoudbilal explained me recently some slightly counterintuitive convention.

Reply:
@mwuethri  this computes the forward kinematics of all the frames and stores them in data.oMf , I think your confusion is regarding the jacobians

Reply:
right, then it's all fine.